test/unit/math/prim/mat/prob/normal_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <boost/math/distributions.hpp>
+#include <boost/random/mersenne_twister.hpp>
+#include <test/unit/math/prim/mat/prob/vector_rng_test_helper.hpp>
+
+using Eigen::Dynamic;
+using Eigen::Matrix;
+
+struct dist_wrapper {
+  template<typename T1, typename T2, typename T3>
+  auto operator()(const T1& a1, const T2& a2, T3& a3) const {
+    return stan::math::normal_rng(a1, a2, a3);
+  }
+};
+
+std::vector<double> build_quantiles(int N, double mu, double sigma) {
```

Comment:
This is just code I copied from the scalar test: https://github.com/stan-dev/math/blob/develop/test/unit/math/prim/scal/prob/normal_test.cpp#L29

It's a callable so I can pass it into the other test functions